src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
> It makes sense to send every application-level error to NewRelic even if the error has nothing to do with NewRelic itself?

Yes, New Relic is a monitoring and analytics tool. They provide error analytics, among other things. So one can look at [pretty graphs](http://blog.newrelic.com/wp-content/uploads/APM-error-analytics-2.jpg) about which hosts suffer how many errors of this or that class at this or that time of the day, during this or that transaction/request/job.

> It makes sense to send only error-level information; there's no analog for other logging levels?

As far as I can tell, there are no other log levels (I am no expert, we only started using this thing).